src/App.tsx

```diff
@@ -455,10 +455,12 @@ const App: React.FC = () => {
             name: 'write-in',
             partyId: ''
           } as CandidateInterface
```

Comment:
FWIW, this should be `const writeIn: CandidateInterface = {` and not `} as CandidateInterface` because the latter will allow “close enough” objects that don’t actually type-check strictly.

Reply:
Thanks @eventualbuddha. Will fix. Shared wisdom and experience always welcome.